docsp-29311 - upcoming breaking changes (#374)
mongodb/docs-java: source/whats-new.txt

  and map-reduce methods on `MongoCollection <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#mapReduce(com.mongodb.client.ClientSession,java.lang.String,java.lang.String)>`__
  are deprecated in MongoDB v4.2 and later. They will be replaced by the
  aggregation framework and removed in a future release.

> - The {+driver-short+} v5.0 and later will require Java 11 or later.

Reviewer: [suggestion]

Minor nitpick - the repeat of "and later...or later" in close proximity reads a little odd to me. Maybe consider changing the wording a little?

```suggestion
- Starting with {+driver-short+} v5.0, you will need to use Java 11 or later.
```

Author: That's how I would have written it too--I agree on the weirdness--but Chris pointed me to a [style guide section](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#:~:text=To%20refer%20to%20releases%20or%20product%20versions%2C%20use%20later) that indirectly endorses the "version x and later" pattern.

That said, since you agree with my initial instinct, I'll change this occurrence.
